mautrix_telegram/example-config.yaml

```diff
@@ -99,19 +99,57 @@ metrics:
 telemetry:
```

Comment:
Does telemetry transmission period need to be [configurable](https://gitlab.matrix.org/new-vector/modular/synapse-telemetry/-/blob/254d208f7a82919240659e790a794d404631c561/README.md#L74), or is that handled by something else?

Reply:
Not for this bridge, I don't think.  Since the only telemetry gathered is RMAU, telemetry is only sent 1) on startup 2) when the RMAU count changes (tracked by the pre-existing metric callback of [`TelegramBridge._update_active_puppet_metric`](https://github.com/vector-im/mautrix-telegram/blob/element-master/mautrix_telegram/__main__.py#L229)).